katello/katello/hostname-change.rb

```diff
@@ -211,6 +233,22 @@ def run
 
       self.precheck
 
+      scenario_answers = load_scenario_answers(@options[:scenario])
+
+      unless custom_cert_options_check(scenario_answers)
+        fail_with_message("You are currently using custom certificates but not all custom certificate arguments are present. To change the hostname, " \
+                          "you need to provide certificates with the new hostname. Please review the custom certificate arguments required with --help.")
+      end
+
+      if @using_custom_certs && !@foreman_proxy_content
+        STDOUT.puts "\nChecking custom certificates"
+        [@options[:custom_cert], @options[:custom_key], @options[:custom_cert_req]].each do |file|
+          check_if_file_exists(file)
```

Comment:
Would it be nice if we checked all files and failed with one message that some files don't exist?